cri/v1alpha2/cri_utils.go

```diff
@@ -1087,6 +1087,22 @@ func parseVolumesFromPouch(containerVolumes map[string]interface{}) map[string]*
 	return volumes
 }
 
+// parseEnvsFromPouch parse Envs from []string to []*runtime.KeyValue
+func parseEnvsFromPouch(pouchEnvs []string) (criEnvs []*runtime.KeyValue) {
+	for _, env := range pouchEnvs {
+		runtimeEnv := &runtime.KeyValue{}
+		if strings.Count(env, "=") == 1 {
+			envItem := strings.Split(env, "=")
```

Comment:
What will happen if `env` is equal to `a=b=c`?

Reply:
@allencloud Actually the envs here is returned from pouchd and they should have been validated when input to pouchd.

So maybe we don't need to check again here.

Reply:
Yeah, in fact, the case 'a=b=c' will not be passed by pouchd. 
Even if that happens, the upper level can also deal with it by key.

Reply:
But you are using `envItem[1]` to take `b` and ignoring `=c`, right? @Starnop

Reply:
`a=b=c` is valid because `b=c` is value. 

```
# vagrant @ ubuntu-xenial in ~ [20:36:09]
$ A=1=2 echo ${A}
1=2
```

Reply:
I sponsor @fuweid 's point of view. Your split way has potential risk. Do you agree? @Starnop